proposals/NNNN-Add-a-mechanism-to-exclude-route-guidance-between-embedded-navigation-and-SDL-navigation.md

```diff
@@ -0,0 +1,138 @@
+# Add a mechanism to exclude route guidance between embedded navigation and SDL navigation
+
+* Proposal: [SDL-NNNN](NNNN-Add-a-mechanism-to-exclude-route-guidance-between-embedded-navigation-and-SDL-navigation.md)
+* Author: [Shohei Kawano](https://github.com/Shohei-Kawano)
+* Status: Awating Review
+* Impacted Platforms: [Core / HMI / iOS / Java Suite / RPC]
+
+## Introduction
+
+Add exclusive control mechanism to prevent conflict of route guidance between embedded navigation and SDL navigation.  
```

Comment:
```suggestion
This proposal is to add an exclusive control mechanism to prevent conflict of route guidance between embedded navigation and SDL navigation.  
```